mods/raclassic/rules/husks.yaml

```diff
@@ -69,6 +87,51 @@ MCV.Husk:
 	RenderSprites:
 		Image: mcvhusk
 
+JEEP.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Ranger)
+	TransformOnCapture:
+		IntoActor: jeep
+	RenderSprites:
+		Image: jeep.destroyed
+
+APC.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Armored Personnel Carrier)
+	TransformOnCapture:
+		IntoActor: apc
+	RenderSprites:
+		Image: apc.destroyed
+
+MNLYAP.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Mine Layer)
+	TransformOnCapture:
+		IntoActor: mnly.ap
+	RenderSprites:
+		Image: mnly.destroyed
+
+MNLYAV.Husk
+	Inherits: ^Husk
+	Tooltip:
+		Name: Husk (Mine Layer)
+	TransformOnCapture:
+		IntoActor: mnly.av
+	RenderSprites:
+		Image: mnly.destroyed
+
+TRUK.Husk
```

Comment:
I'm not sure how does this even works, you are misssing the colon on all the new husk names.